iron-ajax.html

```diff
@@ -291,7 +291,14 @@
     },
 
     get requestHeaders() {
-      var headers = Object.create(this.headers || {});
+      var headers = {},
+          header;
+
+      if(this.headers instanceof Object) {
```

Comment:
It will probably be sufficient to check for the existence of `this.headers`. Checking that it with `instanceof` doesn't give us many guarantees about the type anyway.